docs/detections/rules-ui-create.asciidoc

```diff
@@ -114,75 +125,137 @@ image::images/rule-query-example.png[]
 TIP: This example is based on the
 <<volume-shadow-copy-deletion-via-vssadmin, Volume Shadow Copy Deletion via VssAdmin>> prebuilt rule.
 
-* To create a rule based on a source event field threshold, select _Threshold_
+* To create a rule based on a source event field threshold, select *Threshold*
 and then:
 .. Define which {es} indices the rule analyzes for alerts.
 .. Use the filter and query fields to create the criteria used for detecting
 alerts.
-.. Use the _Field_ and _Threshold_ fields to determine which source event field
+.. Use the `Field` and `Threshold` fields to determine which source event field
 is used as a threshold and the threshold's value.
 +
-For example, if the _Field_ is `source.ip` and its _Threshold_ is `10`, an
+For example, if the `Field` is `source.ip` and its `Threshold` is `10`, an
 alert is generated for every source IP address that appears in at least 10 of
 the rule's search results.
 +
-You can also leave the _Field_ undefined. The rule then creates an alert when
-the number of search results is equal to or greater than the _Threshold_ value.
+You can also leave the `Field` undefined. The rule then creates an alert when
+the number of search results is equal to or greater than the `Threshold` value.
 +
 If you want an alert for 10 or more failed login attempts to a specific host
-per rule execution:
+per rule execution, enter the following rule parameters:
+
+** *Custom query*: `host.name : liv-win-19 and event.category : "authentication" and event.outcome : "failure"`
+** *Field*: Leave blank
+** *Threshold*: `10`
+
+* To create an event correlation rule using Event Query Language, select *Event Correlation* and then:
+.. Define which {es} indices the rule searches for alerts.
+.. Add an {ref}/eql-syntax.html[EQL statement] used to detect alerts.
+
+For example, the following rule detects when `msxsl.exe` makes an outbound
```

Comment:
The indentation of this section appears to be a bit off.

![image](https://user-images.githubusercontent.com/616158/97877890-db798a80-1ceb-11eb-9fed-95dace7a4954.png)